ckan/authz.py

```diff
@@ -102,13 +102,19 @@ def _build(self):
                     fetched_auth_functions[name] = auth_function
 
         for name, func_list in chained_auth_functions.iteritems():
-            if name not in fetched_auth_functions:
+            if name not in fetched_auth_functions and\
```

Comment:
Do you mind using parentheses instead of `\ ` to wrap the line? Just to be consistent with the styling we use elsewhere in the code.